src/inspector/node_protocol.pdl

```diff
@@ -37,3 +37,59 @@ experimental domain NodeTracing
   # Signals that tracing is stopped and there is no trace buffers pending flush, all data were
   # delivered via dataCollected events.
   event tracingComplete
+
+# Supports additional targets discovery and allows to attach to them.
+experimental domain NodeTarget
```

Comment:
After some discussion, we think that we'll not be able to keep NodeTarget domain close to Chromium's Target domain on the frontend side in the long term. Therefore, it would be easier to keep this domain very node-worker-specific to reduce the complexity and maintenance for the Node community.

My proposal would be to just rename things, keeping semantics the same:
- NodeTarget -> NodeWorker
- TargetID -> WorkerID
- TargetInfo -> WorkerInfo
- sendMessageToTarget -> sendMessageToWorker
- setAutoAttach(true, waitForDebuggerOnStart) -> enable(waitForDebuggerOnStart)
- setAutoAttach(false) -> disable()
- attachedToTarget -> workerCreated
- detachedFromTarget -> workerDestroyed
- receivedMessageFromTarget -> receivedMessageFromWorker

What do you think? Other than that, things look good to me.

Reply:
Sounds like a plan! I will implement the changes.

Reply:
Did that. One thing I kinda feel strongly is that worker(Created|Destroyed) would be a misnomer, so I renamed it to attachedToWorker/detachedFromWorker. I would also consider naming them simply "attached" or "detached" as they are already have NodeWorker prefix.

Reply:
Sounds good, I like "attachedToWorker/detachedFromWorker".